micrometer/prometheus-kafka/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,34 @@
+####
```

Comment:
Please drop `docker/Dockerfile.*`, they are outdated and we do not need them

Reply:
Done